index.js

```diff
@@ -61,16 +67,20 @@ var showQuestions = function() {
     questionsPageElement.show();
 };
 
-var resetScore = function() {
-    scoreElement.text(0);
+var resetState = function(state, element) {
+    state.current = 0;
+    state.score = 0;
+    element.text(state.score);
 };
 
-var increaseScore = function() {
-    var score = parseInt(scoreElement.text(), 10);
-    scoreElement.text(score + 1);
+var increaseScore = function(state, element) {
+    state.score += 1; 
+    element.text(state.score);
 };
 
-var setQuestion = function(questionIndex) {
+
+var setQuestion = function(state) {
```

Comment:
This is sort of a *render* type function but it doesn't receive an element.  It's using global variables -- we need to remove all use of global variables in our functions. The function can only use the params it is passed.